src/lib/tests/import-export/import/handlers/backup-handler.test.ts

```diff
@@ -0,0 +1,124 @@
+// To run tests, execute `npm run test -- src/lib/tests/import-export/import/handlers/backup-handler.test.ts``
+import fs from 'fs';
+import path from 'path';
+import zlib from 'zlib';
+import AdmZip from 'adm-zip';
+import * as tar from 'tar';
+import { BackupHandler } from '../../../../import-export/import/handlers/backup-handler';
```

Comment:
Could we place tests closer to the component we are testing, similarly to other files?

Reply:
I'll move them inside `import` folder. I wanted to have all tests in one place ( related to `lib` ), but I get your point